docs/contributing/ci/runners/windows-vm/index.rst

```diff
@@ -0,0 +1,330 @@
+    PasswordAuthentication yes
+    
+- Restart the sshd service.
+    
+.. code-block:: console
+
+    Restart-Service sshd
+    
+
+VirtIO drivers:
+
+- Find the latest stable `VirtIO`_ driver as a disk image, probably called something 
+  like ``virtio-win.iso``.
+
+- Open the downloaded ISO, and run the ``virtio-win-guest-tools.exe`` installer.
+
+- Click next through it all to install it.
+
+Enable WMIC:
+
+- Open a Terminal as Administrator and run the following command (takes a while).
+    
+.. code-block:: console
+
+  Add-WindowsCapability -Online -Name WMIC
+
+Check
+~~~~~
+
+Test whether the SSH server has been configured correctly:
+
+- Inside the Windows VM, run ``ipconfig`` to find the IPv4 Address.
+
+- On the host machine, run ``ssh root@<ipv4>`` with password ``root`` to see that you can
+  ssh into the machine and it starts a msys2 shell.
+
+- Close down the VM through the GUI and close virt-manager.
+
+You can also follow the guide below to test that you can start a QEMU guest manually with the qcow2 image.
+
+Upload the qcow image to the Hetzner storage box.
+
+.. code-block:: bash
+
+  scp /var/lib/libvirt/images/<qcow image> <storage-box url>:~/win11.qcow2
+
+
+Starting QEMU with qcow2 image
+------------------------------
+
+Assuming you want to run qemu as user ``ghr`` with home directory ``/home/ghr``. This
+might be necessary to setup as ``root``.
+
+Setup home directory
+~~~~~~~~~~~~~~~~~~~~
+
+Create subdirectories:
+
+.. code-block:: bash
+
+  mkdir -p /home/ghr/system_imaging
+  cd /home/ghr/system_imaging
+  mkdir -p disk fd iso
+
+
+Get the qcow2 image:
+
+- If created on this machine, copy from `libvirt` directory:
+    
+.. code-block:: bash
+    
+  cp /var/lib/libvirt/images/<qcow image> /home/ghr/system_imaging/disk
+    
+- If not, copy from Hetzner storage box:
+    
+.. code-block:: bash
+
+  scp <storage-box url>:~/win11.qcow2 /home/ghr/system_imaging/disk
+    
+
+Get drivers:
+
+- Copy the Windows BIOS drivers to the `fd` directory
+    
+.. code-block:: bash
+
+  cp /usr/share/OVMF/OVMF_CODE_4M.ms.fd /home/ghr/system_imaging/fd/win11_VARS.fd
+    
+- Download the VirtIO driver to the `iso` directory
+    
+.. code-block:: bash
+
+  cd /home/ghr/system_imaging/iso && wget https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/virtio-win-0.1.271-1/virtio-win.iso
+
+Change ownership:
+
+.. code-block:: bash
+
+  chown -R ghr:ghr /home/ghr/system_imaging
+
+Check
+~~~~~
+
+Login as user ``ghr``.
+
+.. code-block:: bash
+
+  su - ghr
+
+Test with QEMU:
+
+- Copy the original image to somewhere safe.
+    
+.. code-block:: bash
+
+  mkdir -p /home/ghr/guests/tmp
+  rm -rf /home/ghr/guests/tmp/*
+  cp /var/lib/libvirt/images/<qcow2 image> /home/ghr/guests/tmp/boot.img
+
+- Start a swtpm socket
+    
+.. code-block:: bash
+
+  swtpm socket --tpm2 --ctrl type=unixio,path=/home/ghr/guests/tmp/swtpm.sock --tpmstate dir=/home/ghr/guests/tmp --flags not-need-init --daemon
```

Comment:
When the host is rebooted  must this be started manually again?

Reply:
Yes, this is a step that must be done before running the VM - which is also why it's under the "Test with QEMU" header. In our CI, it's part of the `windows_qemu_prep.py` script